data/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "data",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "devDependencies": {
```

Comment:
Should the analysis be a separate node module or should we add these two `devDependencies` to the top level package.json?